engine/common/synchronization/engine.go

```diff
@@ -582,16 +597,16 @@ func (e *Engine) onBatchRequest(originID flow.Identifier, req *messages.BatchReq
 	}
 	err := e.con.Unicast(res, originID)
 	if err != nil {
-		return fmt.Errorf("could not send batch response: %w", err)
+		e.log.Warn().Err(err).Msg("sending batch response failed")
+		return nil
 	}
-
 	e.metrics.MessageSent(metrics.EngineSynchronization, metrics.MessageBlockResponse)
 
 	return nil
 }
 
 // onBlockResponse processes a response containing a specifically requested block.
-func (e *Engine) onBlockResponse(originID flow.Identifier, res *messages.BlockResponse) {
+func (e *Engine) onBlockResponse(originID flow.Identifier, res *messages.BlockResponse) error {
```

Comment:
Why returning error if the error is always nil

Reply:
👍 cleaned up in https://github.com/onflow/flow-go/pull/1008/commits/cabc8ac7d0156cda146511b1733a0a3a020a54ed